cmd/tfsec-docs/navigation_docs.go

```diff
@@ -16,6 +16,7 @@ const navDocsTemplate = `---
   state: open
   docs:
     - installation
+	- usage
```

Comment:
This indenting is weird, or the github app is weird, can't tell which.

Reply:
it was something funky in my ide masking the indent